sdk/rust/nym-sdk/examples/tcp_proxy_multistream.rs

```diff
@@ -0,0 +1,142 @@
+use bincode;
+use nym_sdk::mixnet::Recipient;
+use nym_sdk::tcp_proxy;
+use rand::rngs::SmallRng;
+use rand::Rng;
+use rand::SeedableRng;
+use serde::{Deserialize, Serialize};
+use tokio::io::AsyncWriteExt;
+use tokio::net::TcpStream;
+use tokio::signal;
+use tokio::task;
+use tokio_stream::StreamExt;
+use tokio_util::codec;
+use tracing_subscriber;
+
+#[derive(Serialize, Deserialize, Debug)]
+struct ExampleMessage {
+    message_id: i8,
+    message_bytes: Vec<u8>,
+    tcp_conn: i8,
+}
+
+// This example just starts off a bunch of Tcp connections on a loop to a remote endpoint: in this case the TcpListener behind the NymProxyServer instance on the echo server found in `nym/tools/echo-server/`. It pipes a few messages to it, logs the replies, and keeps track of the number of replies received per connection.
+//
+// To run:
+// - run the echo server with `cargo run`
+// - run this example with `cargo run --example tcp_proxy_multistream
+#[tokio::main]
+async fn main() {
```

Comment:
Still liking the `anyhow` :)

```suggestion
async fn main() -> anyhow::Result<()> {
```